bundles/binding/org.openhab.binding.girahomeserver/src/main/java/org/openhab/binding/girahomeserver/internal/GiraHomeServerBinding.java

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.girahomeserver.internal;
+
+import java.util.Map;
+
+import org.openhab.binding.girahomeserver.GiraHomeServerBindingProvider;
+
+import org.apache.commons.lang.StringUtils;
+import org.openhab.core.binding.AbstractActiveBinding;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.osgi.framework.BundleContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+	
+
+/**
+ * Implement this class if you are going create an actively polling service
+ * like querying a Website/Device.
+ * 
+ * @author Jochen Mattes
+ * @since 1.8.0-SNAPSHOT
+ */
+public class GiraHomeServerBinding extends AbstractActiveBinding<GiraHomeServerBindingProvider> {
+
+	private static final Logger logger = 
+		LoggerFactory.getLogger(GiraHomeServerBinding.class);
```

Comment:
Please use the project formatting and cleanup rules in the Eclipse IDE -- this line looks like it wraps earlier than the formatter would do?